infra-as-code/bicep/modules/mgDiagSettings/README.md

```diff
@@ -0,0 +1,28 @@
+# Module: Enable Diagnostic Settings on Management Groups
+
+This module enables Diagnostic Settings on a management group to an existing Azure Log Analytics Workspace, normally deployed during the logging.bicep module.
+
+## Parameters
+
+The module requires the following input parameters. This parameter is passed from the mgDiagSettingsAll.bice module.
+
+| Parameter                             | Type   | Description                                                                                                                                                                          | Requirements                      | Example                                                                                 |
+| ------------------------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------- | --------------------------------------------------------------------------------------- |
+| parLawId    | string   | Id of the Log Analytics Workspace                                                             | Mandatory input, default: `false` | `false`                                                                                 |
+
+## Outputs
+
+The module will not generate any outputs.
+
+## Deployment
+
+This module requires being called with the  scope: managementGroup(mgId) function. This is done from within the orchestration module mgDiagSettingsAll.bicep module for all the management groups previously created and based on the input parameters for that parent module.
+The Diagnostic Settings resource will be named toLa but can be changed in the module if desired.
+
+## Validation
+
+To validate if Diagnostic Settings was correctly enabled for any specific management group, a REST API GET call can be used. Documentation and easy way to try this can be found in this link [(Management Group Diagnostic Settings - Get)](https://learn.microsoft.com/rest/api/monitor/management-group-diagnostic-settings/get?tabs=HTTP&tryIt=true&source=docs#code-try-0).
```

Comment:
Add a note here as to why this is required. Because the portal experience does not show these 👍